web/components/NavLinkItem.js

```diff
@@ -3,37 +3,58 @@ import { Link } from 'react-router-dom'
 import Box from './Box'
 import Flex from './Flex'
 import Text from './Text'
+import Icon from './Icon';
+import Card from './Card';
 
-export default ({ text, selected, icon, route }) => (
-    <Link to={route}>
-        <Flex
-            nowrap
-            color={selected ? 'blue' : 'charcoal'}
-            height={60}
+export default ({
+    text, selected, icon, route, width
+}) => (
+    <Box
+        display='inline-flex'
+        width={width + '%'}
+    >
+        <Card
+            onClick={() => {
+            }}
+            bg={selected ? 'black' : 'grey'}
+            width='100%'
+            borderLeft='2px solid black'
+            borderRight='2px solid black'
         >
-            <Box
-                width='4px'
-                height='100%'
-                bg={selected ? 'blue' : 'transparent'}
-            />
-            <Flex
-                direction={['column', 'column', 'row']}
-                alignItems='center'
-                pl={[0, 0, '1rem']}
-                flex={1}
+            <Link
+                to={route}
+                style={{ textDecoration: 'none' }}
             >
-                {React.createElement(icon, {
-                    size: 24
-                })}
-                <Text
-                    color={selected ? 'blue' : 'charcoal'}
-                    mt={['4px', '4px', 0]}
-                    ml={[0, 0, '1rem']}
-                    size={[10, 10, 14]}
+                <Card
+                    bg='black'
+                    mx='50%'
+                    mt='5%'
+                    depth='-2px 0px 10px rgba(0,0,0,0.5)'
                 >
-                    {text}
-                </Text>
-            </Flex>
-        </Flex>
-    </Link>
+                    <Flex
+                        direction={['column', 'column', 'row']}
+                        alignItems='center'
+                        pl={[0, 0, '1rem']}
+                        flex={1}
+                    >
+                        <Icon
+                            faIconName={icon} 
+                            size='5x'
+                            color={selected ? 'white' : 'charcoal'}
+                        />
+                        <Text
+                            color={selected ? 'white' : 'charcoal'}
+                            mt={['4px', '4px', 0]}
+                            ml={[0, 0, '1rem']}
+                            size={[10, 10, 14]}
+                            fontFamily='Nunito,Helvetica Neue,Helvetica,Arial,sans-serif'
```

Comment:
Do we need this line if we can just change the defaultProps on the Text component?